src/plugins/default/DuplicateCheckerPlugin.ts

```diff
@@ -1,50 +1,70 @@
 import { IInnerError } from '../../models/IInnerError';
-import { ILog } from '../../logging/ILog';
 import { IEventPlugin } from '../IEventPlugin';
 import { EventPluginContext } from '../EventPluginContext';
 import { Utils } from '../../Utils';
 
 export class DuplicateCheckerPlugin implements IEventPlugin {
-  public priority: number = 40;
+  public priority: number = 90;
```

Comment:
The priority should be really really high, in .net we do 1010 to ensure it runs last.